sklearn/cross_decomposition/_pls_.py

```diff
@@ -420,6 +420,32 @@ def transform(self, X, Y=None, copy=True):
 
         return x_scores
 
+    def inverse_transform(self, X, copy=True):
+        """Transform data back to its original space.
+        Parameters
+        ----------
+        X : array-like, shape (n_samples, n_components)
+            New data, where n_samples is the number of samples
+            and n_components is the number of pls components.
+        copy : boolean, default True
+            Whether to copy X, or perform in-place normalization.
+        Returns
+        -------
+        X_original array-like, shape (n_samples, n_features)
```

Comment:
```suggestion
        X_original: array-like of shape (n_samples, n_features)
```

Reply:
I just pushed all the docstring changes. Thanks!